### PR TITLE
Changes.

### PR DESCRIPTION
**Pull request by antim0118**
tested on:
- CS:GO - de_mirage
- CS:S - test_speakers, cs_compound
- The Stanley Parable - map

### CODE REVIEW

**Comment by Fronkln** [comment]
Interestingly enough i have checked if the code at ??? works before putting it here

File: Assets/Code/ConvertUtils.cs [hunk 1]
```diff
@@ -113,6 +113,8 @@ public static float[] Vector3ArrayToFloatArray(Vector3[] vecIn)
 
 			return fOut;
 		}
+
+        public static float floatParse(string str) => float.Parse(str.Replace(".", ","));
```
Comment (lewa-j) [inline Assets/Code/ConvertUtils.cs]: Это костыль. Тут решение получше https://stackoverflow.com/questions/1014535/float-parse-doesnt-work-the-way-i-wanted

**Comment by antim0118** [comment]
> Interestingly enough i have checked if the code at ??? works before putting it here

Unity and VS throws exception.

**Comment by Fronkln** [comment]
I don't think that matters now considering it's been fixed, just gotta have lewa commit it

**Comment by antim0118** [comment]
> I don't think that matters now considering it's been fixed, just gotta have lewa commit it

Unity-Source-Tools/Assets/Code/Test.cs : line 41
`public string mapName;`
"**m**apName", not "**M**apName"

edit: you replied faster than i edited last comment

File: Assets/Code/ConvertUtils.cs (hunk 1 above)
Comment (antim0118) [reply Assets/Code/ConvertUtils.cs]: Исправил.

**Pull request merged**